### PR TITLE
[To new_cluster] Resolve conversation for new cluster

### PR DESCRIPTION
**Pull request by CRZbulabula**

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java [hunk 1]
```diff
@@ -47,6 +47,7 @@
 
   private LoadBalancer loadBalancer;
 
+  /** Test only constructor */
   public ConfigManager(String hashExecutorClass, int deviceGroupCount) {
```
Comment (wangchao316) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:51]: if a method only for test,  need add @TestOnly

**Review by wangchao316 — CHANGES_REQUESTED** [review]

**Review by qiaojialin — APPROVED** [review]

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java (hunk 1 above)
Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:51]: I found that @TestOnly is in org.apache.iotdb.db.utils.TestOnly package. But I couldn't dependence iotdb-server.

Comment (CRZbulabula) [reply confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:53]: Fix it

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**